api/file_manager/file_manager.ts

```diff
@@ -12,7 +12,10 @@ export async function createFile(
     fileStream: fs.ReadStream
 ): Promise<string> {
     const JWTAuth: JWT = await getJWTAuth()
-    const drive = google.drive({ version: 'v3', auth: JWTAuth })
+    const drive: drive_v3.Drive = google.drive({
+        version: 'v3', 
+        auth: JWTAuth
+    })
```

Comment:
I'm not sure you need to define the drive every time you do an action, why not create it globally and use it when you need it